src/scripts/coffeemate.test.js

```diff
@@ -69,6 +67,45 @@ describe("coffeemate", () => {
       });
     });
 
+    it("sends an ephemeral message the first time a user asks for a coffee in a different scope", async () => {
```

Comment:
these tests are great! thank you!

i think there are probably some opportunities to structure them a little better

```
describe('the first time a user asks for coffee', () => {
  describe('without a scope, async () => {
	...
  )}
  describe('with a scope', () => {
	describe('please is the scope', async () => {
		...
	}
	describe('an arbitrary scope', async () => {
		...
	})
  })
```

this way you could probably share some expectations in a before block, and they test results will probably be more readable. 

there may be other, better ways to structure this, like by scope rather than times it's been called! and also these tests are fine the way they are, this is definitely just a suggestion. (i still have designs on finding some time to help restructure these)